src/librustdoc/clean/inline.rs

```diff
@@ -104,7 +104,11 @@ pub fn try_inline(cx: &DocContext, def: Def, name: ast::Name, visited: &mut FxHa
         // separately
         Def::Macro(did, MacroKind::Bang) => {
             record_extern_fqn(cx, did, clean::TypeKind::Macro);
-            clean::MacroItem(build_macro(cx, did, name))
+            if let Some(mac) = build_macro(cx, did, name) {
```

Comment:
Out of curiosity, is there a reason why it cannot be `clean::MacroItem(build_macro(cx, did, name)?);`

Reply:
Because i never use the question mark operator, so i never thought about it! 😅